src/js/BookReader/options.js

```diff
@@ -202,6 +202,8 @@ export const DEFAULT_OPTIONS = {
       className: 'thumb',
     },
   },
+
+  startFullscreen: false,
```

Comment:
Suggestion: add the docstring

Reply:
This MR was already merged, but a new one for the fix is [here](https://github.com/internetarchive/bookreader/pull/329) and includes a docstring.